openstudiocore/src/model/ElectricEquipment.cpp

```diff
@@ -346,6 +348,17 @@ ElectricEquipment::ElectricEquipment(const ElectricEquipmentDefinition& electric
   : SpaceLoadInstance(ElectricEquipment::iddObjectType(),electricEquipmentDefinition)
 {
   OS_ASSERT(getImpl<detail::ElectricEquipment_Impl>());
+
+  /*
+   *Schedule sch = this->model().alwaysOnDiscreteSchedule();
+   *bool test = this->setSchedule(sch);
+   *OS_ASSERT(test);
+   *test = this->setMultiplier(1.0);
+   *OS_ASSERT(test);
+   */
+  bool test = this->setEndUseSubcategory("General");
```

Comment:
@jmarrec this is already the default so don't really need to set this here?

 ```
 A6; \field End-Use Subcategory
       \type alpha
       \retaincase
       \default General
```

Reply:
Oh I get it, this was to force the inspector to have this number of fields.  I guess you could have set it to "" but this is better